### ISSUE TITLE
Enum example

### ISSUE THREAD

**Issue by bander-saeed94** [halvardssm/deno-nessie#35]
It would a time-saver to add more examples. It took me some time to figure out how to make an enum.

I had to check how to do it in PostgreSQL and find out what is wrong.

Before:
```
import { Schema } from "https://deno.land/x/nessie/mod.ts";

export const up = (schema: Schema): void => {
  schema.create("tbl_deposit", (table) => {
    ...
    table.enum(
      "deposit_type",
      ["monthlyContribution", "monthlyInstallment"],
    );
   ...
    console.log(table.toSql());
  });
};

export const down = (schema: Schema): void => {
  schema.drop("tbl_deposit");
  schema.queryString("DROP TYPE deposit_type;");
};
```

So I had to make a single quote string inside a double-quoted string 
After:
```
import { Schema } from "https://deno.land/x/nessie/mod.ts";

export const up = (schema: Schema): void => {
  schema.create("tbl_deposit", (table) => {
    ...
    table.enum(
      "deposit_type",
      ["'monthlyContribution'", "'monthlyInstallment'"],
    );
   ...
    console.log(table.toSql());
  });
};

export const down = (schema: Schema): void => {
  schema.drop("tbl_deposit");
  schema.queryString("DROP TYPE deposit_type;");
};
```

**Comment by halvardssm** [comment]
Hi @bander-saeed94 , thanks for reporting this bug! I Right now, I am transitioning to a modular structure. As soon as #34 is complete (today hopefully), I will go over the query builder and fix all those small bugs 👍

**Comment by halvardssm** [comment]
@bander-saeed94 Enums were fixed in #34 , you can use enums without `'` now 💪
